GDJS/Runtime/pixi-renderers/pixi-object-effects-manager.ts

```diff
@@ -0,0 +1,271 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2013-2021 Florian Rival (Florian.Rival@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+namespace gdjs {
+  import PIXI = GlobalPIXIModule.PIXI;
+
+  type RendererEffects = Record<string, PixiFiltersTools.Filter>;
+
+  export interface EffectsTarget {
+    getRuntimeScene: () => RuntimeScene;
+    getElapsedTime: (runtimeScene: RuntimeScene) => number;
+    getHeight: () => number;
+    getWidth: () => number; 
+  };
+
+  class PixiObjectEffectsManager {
+    /**
+     * Initialize the renderer effects from the effect data.
+     * @param effectData The effect data
```

Comment:
Describe what is needed to do after the initialization (i.e: call applyEffect to actually add the effect to the renderer object).